samples/sid_end_device/sample.yaml

```diff
@@ -1,113 +1,63 @@
 sample:
   name: Sidewalk end device sample
   description: Sample implementing Amazon Sidewalk End Device
+common:
+  build_only: true
+  platform_allow:
+    - nrf52840dk_nrf52840
+    - nrf5340dk_nrf5340_cpuapp
+    - nrf54l15pdk_nrf54l15_cpuapp
+  integration_platforms:
+    - nrf52840dk_nrf52840
+    - nrf5340dk_nrf5340_cpuapp
+    - nrf54l15pdk_nrf54l15_cpuapp
 tests:
   sample.sidewalk.hello:
-    build_only: true
-    platform_allow:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     extra_configs:
       - CONFIG_SID_END_DEVICE_PERSISTENT_LINK_MASK=y
-    integration_platforms:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     tags: Sidewalk hello
 
   sample.sidewalk.hello.release:
-    build_only: true
-    platform_allow:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     extra_args:
       CONF_FILE=prj_release.conf
```

Comment:
would change to `FILE_SUFFIX=release` though note difference in child/parent image and sysbuild:

- child/parent: `FILE_SUFFIX` only applies to main image, you need to repeat it multiple times with image name to apply to other images e.g. `mcuboot_FILE_SUFFIX`, `b0_FILE_SUFFIX`
- sysbuild: `FILE_SUFFIX` applies to **all** images, to only apply to the main image use `<app>_FILE_SUFFIX` where `<app>` is name of the app, generally the folder name

Reply:
Don't get it, in sysbuild does the file_suffix apply to all images or only to main image?

Reply:
sysbuild: `-DFILE_SUFFIX` applies to all, `-Dmcuboot_FILE_SUFFIX` only applies to mcuboot, `-D<app_name>_FILE_SUFFIX` only applies to `<app_name>` image